Remove references of Mpin.h

avlaev opened the pull request.

tatiNo5 commented:
It gives me warnings on building

tatiNo5 commented:
Could you also change the name of the target and the project to be MfaSDK instead of MPinSDK?

tatiNo5 commented:
MpinCoreLib.framework could stay as a name as we're going to remove it sooner or later :)

tatiNo5 commented:
Remove the `iOS SDK API for Milagro (MPin class)` section form the readme too

avlaev commented:
The project is renamed as well as some of the directories. Warnings are removed(though this is also part of another PR). README has been updated as well.

avlaev commented:
I removed several files which are autogenerated and can be ignored from git.

tatiNo5 commented:
Please, remove the last line of the readme as the link is not open any more

tatiNo5 commented on README.md:
```diff
@@ -10,251 +11,12 @@
+2. Open `MfaSDK/MfaSDK.xcodeproj`
 3. Select *Product->Build* from the Xcode menu.
 
 For further details, see [MIRACL MFA Mobile SDK for iOS Documentation](https://devdocs.trust.miracl.cloud/#/mobile-sdk-instructions/)
```
the link is broken

tatiNo5 commented on README.md:
```diff
@@ -10,251 +11,12 @@
-* `appID` - The _App ID_ used by the backend. The App ID is a unique ID assigned to each customer or application. It is a hex-encoded long numeric value. The App ID can be used only for information purposes and it does not affect the application's behavior in any way.
 
 ## iOS SDK API for MIRACL MFA (`MPinMFA`)
 
```
Please remove the line `It massively resembles the _Apache Milagro_ flavor, while incorporating some functionality is specific to the MIRACL Platform.` and update the next sentence: `Similarly to `MPin`, the `MPinMFA` needs to be initialized.` to be only `The `MPinMFA` object needs to be initialized first.`

tatiNo5 commented on README.md:
```diff
@@ -301,7 +63,7 @@ Identical and analogical to `MPin`'s [`SetBackend`](#mpinstatus-setbackend-const
 ##### `(id<IUser>) MakeNewUser: (const NSString*) identity deviceName: (const NSString*) devName;`
```
Please, copy the description of the methods from the MPIN section you've deleted. Now, the links are wrong and the description is not descriptive

avlaev commented:
I added the changes. I replaced the descriptions for only those methods which have the same signatures as the ones in the removed MPin class. For the others I only removed the broken links.

tatiNo5 commented on README.md:
```diff
@@ -10,46 +11,21 @@
-@property (nonatomic, readwrite) MPinStatus status;
-@property (nonatomic, strong) NSString* errorMessage;
-@property (NS_NONATOMIC_IOSONLY, getter=getStatusCodeAsString, readonly, copy) NSString* statusCodeAsString;
+This flavor of the SDK should be used to build apps that authenticate users against the _MIRACL MFA Platform_.
```
Can `MIRACL MFA Platform.` be a link to the platform... trust.miracl.cloud

tatiNo5 commented on README.md:
````diff
@@ -10,46 +11,21 @@
 
-@end
-```
+The methods that return `MpinStatus`, will always return status `OK` if successful.
````
You could mix this sentence with the previous one: `Most of the methods return a `MpinStatus` object as status `OK` means it passed successfully.`

tatiNo5 commented on README.md:
```diff
@@ -122,10 +101,16 @@ typedef NS_ENUM(NSInteger, UserState)
 
```
MakeNewUser description mention Milagro authentication... it should be Miracl authentication

tatiNo5 commented on README.md:
```diff
@@ -388,33 +192,35 @@ The Platform will also start a user identity verification procedure, by sending
 
 ##### `(MpinStatus*) FinishRegistration: (const id<IUser>) user pin0: (NSString*) pin0 pin1: (NSString*) pin1;`
-This method is generally identical and analogical to `MPin`'s [`FinishRegistration`](#mpinstatus-finishregistration-const-idiuser-user-pin-nsstring-pin),
+This method is generally identical and analogical to `MPin`'s `FinishRegistration`,
```
This description reference to the old mpin method... Correct it

tatiNo5 commented on README.md:
```diff
@@ -388,33 +192,35 @@ The Platform will also start a user identity verification procedure, by sending
 It attempts to retrieve the _Time Permits_ for the user, and if successful, will return Status `OK`.
 If they cannot be retrieved, the method will return Status `REVOKED`.
-If this method is successfully completed, the app should read the PIN/secret from the end-user and call one of the `FinishAuthentication` variants to authenticate the user.
 
```
Why these two sentences are removed? They are essential part of the description

tatiNo5 replied:
The first sentence also needs to be returned back

tatiNo5 commented on README.md:
```diff
@@ -434,7 +240,7 @@ This method authenticates the end-user for logging into a Web App in a browser s
 The `accessCode` associates the authentication with the browser session from which it was obtained.
 
-It is generally identical and analogical to `MPin`'s [`FinishAuthenticationAN`](#mpinstatus-finishauthenticationan-idiuser-user-pin-nsstring-pin-accessnumber-nsstring-an),
+It is generally identical and analogical to `MPin`'s `FinishAuthenticationAN`,
```
It should be MPinMFA, not MPin

avlaev commented:
The changes have been added as well as several other changes which are not listed.

tatiNo5 commented on README.md:
```diff
@@ -532,7 +329,7 @@ The document (`strDoc`) is any free form text that needs to be signed.
 Typically, the customer application will generate a `hash` value for the document that needs to be signed, and will send it to the mobile client app.
 The client app can then verify the correctness of the hash value using this method.
 
-##### `(MpinStatus*) Sign: (id<IUser>) user documentHash: (NSData*) documentHash pin0: (NSString*) pin0 pin1: (NSString*) pin1 epochTime: (double) epochTime authZToken: (NSString*) authZToken result: (BridgeSignature**) result;`
```
The authZToken description below need to be removed too
